src/views/operation/ListSupplies.vue

```diff
@@ -250,8 +257,29 @@ export default {
         this.headers.totalPages = parseInt(response.headers['total-pages']);
         this.headers.currentPage = parseInt(response.headers['current-page']);
         this.headers.totalElements = parseInt(response.headers['total-elements']);
+        this.searchAvailable();
       });
     },
+
+    AsignarDisponibilidad(name, available) {
+      if (this.supplies != null || this.supplies != []) {
+        for (var i = 0; this.supplies.length > i; i++) {
+          if (this.supplies[i].name == name) {
+            this.supplies[i].available = available;
+          }
+        }
+      }
+    },
+    async searchAvailable() {
```

Comment:
esto ta medio raro, por ahi lo que te conviene es usarlo desde el for que los muestra en pantalla y llamarlo para cada uno de los que estas mostrando...
Nada importante, pero puede reducir la carga